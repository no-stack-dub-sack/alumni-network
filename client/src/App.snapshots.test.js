import { LoginSegment } from './components/signup/LoginPage';
import React from 'react';
import renderer from 'react-test-renderer';

/*******************************

  Snapshot Tests!

            Snapshot tests...?

                      ¯\_(ツ)_/¯

*******************************/
//
//
/*******************************
  ./components/signup/LoginPage.js
*******************************/
test('loginSegment renders correctly given "isMobile" is true', () => {
  const loginSegment = renderer.create(<LoginSegment isMobile={true} />);
  expect(loginSegment).toMatchSnapshot();
});

test('loginSegment renders correctly given "isMobile" is false', () => {
  const loginSegment = renderer.create(<LoginSegment isMobile={false} />);
  expect(loginSegment).toMatchSnapshot();
});
