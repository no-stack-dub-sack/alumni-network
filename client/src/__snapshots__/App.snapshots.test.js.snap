exports[`test loginSegment renders correctly given "isMobile" is false 1`] = `
<div
  className="ui segment">
  <h2
    className="ui green image header">
    <i
      className="huge github icon" />
    <div
      className="content">
      Login with GitHub
    </div>
  </h2>
  <div
    className="ui segment">
    <a
      className="ui green button"
      href="//localhost:8080/auth/github">
      Login
    </a>
  </div>
  <div
    className="ui info message">
    <div
      className="header">
      Joining the freeCodeCamp Alumni Network requires
                both freeCodeCamp and GitHub credentials.
    </div>
  </div>
</div>
`;

exports[`test loginSegment renders correctly given "isMobile" is true 1`] = `
<div
  className="ui segment">
  <h2
    className="ui green image header">
    <div
      className="content">
      Login with GitHub
    </div>
  </h2>
  <div
    className="ui segment">
    <a
      className="ui green button"
      href="//localhost:8080/auth/github">
      Login
    </a>
  </div>
  <div
    className="ui info message">
    <div
      className="header">
      Joining the freeCodeCamp Alumni Network requires
                both freeCodeCamp and GitHub credentials.
    </div>
  </div>
</div>
`;
